vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -270,7 +270,21 @@ func (feeder *clusterStateFeeder) InitFromCheckpoints() {
 
 func (feeder *clusterStateFeeder) GarbageCollectCheckpoints() {
 	klog.V(3).Info("Starting garbage collection of checkpoints")
-	feeder.LoadVPAs()
+
+	allVPAKeys := map[model.VpaID]bool{}
+
+	allVpaResources, err := feeder.vpaLister.List(labels.Everything())
+	if err != nil {
+		klog.Errorf("Cannot list VPAs. Reason: %+v", err)
```

Comment:
Can this be changed to structured logging?

Reply:
@adrianmoisey fixed